src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java

```diff
@@ -135,4 +138,32 @@ public void test_compareEQ_Sequential() throws Exception {
 		runTest(new ComparisonTests.TestCompareEQ(),
 				EvaluationStrategy.SEQUENTIAL, StorageStrategy.IN_MEMORY);
 	}
+	
+	@Test
+	public void test_isSorted() throws Exception {
+		runTest(new SortingTests.TestIsSorted(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	
+	@Test
+	public void test_compareAndSwap() throws Exception {
+		runTest(new SortingTests.TestCompareAndSwap(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_Sort() throws Exception {
+		runTest(new SortingTests.TestSort(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	@Ignore
+	public void test_Big_Sort() throws Exception {
+		runTest(new SortingTests.TestBigSort(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_logic() throws Exception {
```

Comment:
I will move it to its own test class.

Logic based on “flags” is likely to be useful in many situations, i think.
Regards
Morten V. Christiansen

Fra: Kasper Damgård <notifications@github.com<mailto:notifications@github.com>>
Svar til: aicis/fresco <reply@reply.github.com<mailto:reply@reply.github.com>>
Dato: onsdag den 24. februar 2016 kl. 23.35
Til: aicis/fresco <fresco@noreply.github.com<mailto:fresco@noreply.github.com>>
Cc: "Morten V. Christiansen" <morten.v.christiansen@alexandra.dk<mailto:morten.v.christiansen@alexandra.dk>>
Emne: Re: [fresco] Sintoperators (#19)


In src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java<https://github.com/aicis/fresco/pull/19#discussion_r54019060>:

> +             runTest(new SortingTests.TestCompareAndSwap(),
> +                             EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
> +     }
> +     @Test
> +     public void test_Sort() throws Exception {
> +             runTest(new SortingTests.TestSort(),
> +                             EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
> +     }
> +     @Test
> +     @Ignore
> +     public void test_Big_Sort() throws Exception {
> +             runTest(new SortingTests.TestBigSort(),
> +                             EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
> +     }
> +     @Test
> +     public void test_logic() throws Exception {


I'm still not sure what the logic thing does :) - What applications does it have?
Anyways, the test should not be within the comparison class. It should have it's own.

—
Reply to this email directly or view it on GitHub<https://github.com/aicis/fresco/pull/19/files#r54019060>.